api-reference/upload-file-api/client-side-file-upload.md

```diff
@@ -344,12 +404,31 @@ import { IKImage, IKContext, IKUpload } from 'imagekitio-react'
 function App() {
   const publicKey = "your_public_api_key";
   let urlEndpoint = "https://ik.imagekit.io/your_imagekit_id";
-  const authenticationEndpoint = "https://www.yourserver.com/auth";
+  const authenticator = async () => {
+    try {
+      // You can also pass headers and validate the request source in the backend, or you can use headers for any other use case.
+      const headers = {
+        'CustomHeader': 'CustomValue'
```

Comment:
Add authorization header in this example as well.

Reply:
done